### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by scottyplunkett**

### CODE REVIEW

**Comment by NicoleCarpenter** [comment]
Why is this 9 files changed for a PR to update the README?

**Comment by scottyplunkett** [comment]
@NicoleCarpenter
2 files were from floobits that were not ignored when they should have been, so the PR deletes them. 
I had 2 makefiles, one in root of repo and one in the app root, one deleted and one was updated 
I had 2 readmes, one in root of repo and one in the app root, one deleted and one was updated 
I deleted a line from gitignore,
I added a runner file to make the instructions for starting a game easier until I turn this into a full OTP release.
They are all small changes, aimed at cleaning up or making the readme easier to follow

**Comment by NicoleCarpenter** [comment]
That's fine, but these changes should not be part of this pull request. 

I am foreshadowing your next project. Commits, commit messages, PR naming,  pull request files changed, diffs, styling... I will care about all of these things.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,36 @@
...
+      ```
+      git clone https://github.com/scottyplunkett/erlang-tic-tac-toe
+      ```
+2.  Change directory into the application's root.
+      ```
+      cd erlang-tic-tac-toe/tictactoe
+      ```
+3.  Compile code and Start the Application Shell.
+      ```
+      make play
```
Comment (NicoleCarpenter) [inline README.md:18]: I got this error:

```
make: *** No rule to make target `play'.  Stop.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,36 @@
...
+      ```
+(Instructions for starting a new tictactoe game will appear when the shell opens.)
+
+
+Running Test:
+
+1.  Fork and clone the repository.
+      ```
+      git clone https://github.com/scottyplunkett/erlang-tic-tac-toe
```
Comment (NicoleCarpenter) [inline README.md:27]: You can omit steps 1 and 2 here for the tests.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,36 @@
...
+      ```
+2.  Change directory into the application's root.
+      ```
+      cd erlang-tic-tac-toe/tictactoe
+      ```
+3.  Compile code and run the test.
+      ```
+      make run_test
```
Comment (NicoleCarpenter) [inline README.md:35]: I got this error:

```
Eshell V8.1  (abort with ^G)
1> undefined
*** error reading file ***
**no such file or directory: ebin

=======================================================
  Failed: 0.  Skipped: 0.  Passed: 0.
One or more tests were cancelled.
```

**Review by NicoleCarpenter — CHANGES_REQUESTED** [review]
Are you missing a build step? I could neither run the applications nor the tests with the README instructions.

**Pull request closed without merging**